sdk/key_vault/src/key.rs

```diff
@@ -239,6 +239,54 @@ impl Display for SignatureAlgorithm {
     }
 }
 
+#[derive(Debug, Deserialize, Getters)]
+#[getset(get = "pub")]
+pub struct DecryptResult {
+    aad: Option<String>,
```

Comment:
This should be `additional_authenticated_data`. See https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/keyvault/Azure.Security.KeyVault.Keys/api/Azure.Security.KeyVault.Keys.netstandard2.0.cs for names, adjusting for idiomatic naming conventions, of course. We want Azure SDKs to be first and foremost idiomatic, but consistent second and naming is one of them.

This also shouldn't be defined on `DecryptResult`. It's passed into `DecryptParameters` (as it should be called). Specifically, see the classes at https://github.com/Azure/azure-sdk-for-net/blob/fe8e623c8ef31810a4712fb12fec4ecbc019859f/sdk/keyvault/Azure.Security.KeyVault.Keys/api/Azure.Security.KeyVault.Keys.netstandard2.0.cs#L395-L422.

Reply:
I was looking at https://docs.microsoft.com/en-us/rest/api/keyvault/decrypt/decrypt#keyoperationresult
and the key operation result contain these field. I guess the azure sdk for .NET is more precise

Reply:
Yeah, we did change a few property names mainly to avoid abbreviations (we have guidelines against that). Abbreviations are great for a transform protocol but not always an SDK, or so the thinking goes. The SDK does also change the shape a bit for some things you haven't touched here, too (so not an issue with this PR).